src/components/cascadeselect/CascadeSelect.d.ts

```diff
@@ -5,11 +5,11 @@ interface CascadeSelectProps {
     style?: object;
     className?: string;
     value?: any;
-    options?: array;
+    options?: Array<any>;
```

Comment:
I believe any[] would fit better with the rest of PrimeReact typing style from what I have seen. Array\<any\> is only used in Timeline.d.ts, any[] is used in 36 places across 21 files.

```suggestion
    options?: any[];
```